modules/rn_3_12_0.adoc

```diff
@@ -30,9 +30,14 @@ The following documentation changes have been made with the {productname} {produ
 
 The following updates have been made to {productname}.
 
-[id=""]
+[id="image-expiration-event"]
+=== Image expiration notification
 
-=== <Example_Feature>
+Previously, {productname} administrators and users had no way of being alerted when an image was about to expire. With this update, an event can be configured to notify users when an image is about to expire, or has expired. This helps {productname} users avoid unexpected pull failures.
```

Comment:
We only track images that are going to expire. We don't notify about images that are already expired.